upgrade/upgrade-scripts/common.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
It looks like the file is already non-executable, which is good. My tendency, and what seems to be the tendency in other Delphix code, is to leave shebangs off of sourced scripts, since they get ignored by the interpreter. If it helps with some editors though, I don't have any objection to leaving it in.

Reply:
That would probably not be a big deal, since we still have the .sh denomination, however I think this would be preferable over specifying the interpreter manually for shellcheck. We also have https://github.com/delphix/appliance-build/issues/186 that may be leveraging this information although I'm not sure.

Reply:
I don't feel very strongly about this, so I've reverted this back to the way it was.